converter/ena/ena_experiment.py

```diff
@@ -1,22 +1,23 @@
 from converter.ena.classes import AttributeType
 from converter.ena.classes.sra_common import PlatformType, RefObjectType, TypeIlluminaModel
 from converter.ena.classes.sra_experiment import Experiment, ExperimentSet, LibraryDescriptorType, LibraryType, SampleDescriptorType, TypeLibrarySelection, TypeLibrarySource, TypeLibraryStrategy
+from converter.ena.ena import EnaModel
```

Comment:
You are not using the `EnaModel` here. I thought you would like to as in `ena-run.py`.

Reply:
Checked. Using EnaModel (renamed from EnaEntity). Discussed @ke4 probably was looking at a different commit.